elastic/kibana
# [Response Ops][Alerting] SLIs Phase 2 - Histogram for task schedule delay

Towards https://github.com/elastic/response-ops-team/issues/130

## Summary

This implements the second of 3 SLIs described in https://github.com/elastic/response-ops-team/issues/130 - the histogram for tracking task schedule delay in seconds. We bucket up to 30 minutes.

## To Verify

Run Kibana and create some alerting rules. Navigate to https://localhost:5601/api/task_manager/metrics?reset=false and you should see that the new metric under `task_run.value.overall.delay`
